common/src/main/java/me/shedaniel/architectury/networking/simple/SimpleNetworkManager.java

```diff
@@ -0,0 +1,36 @@
+package me.shedaniel.architectury.networking.simple;
+
+import me.shedaniel.architectury.networking.NetworkManager;
+import me.shedaniel.architectury.platform.Platform;
+import me.shedaniel.architectury.utils.Env;
+import net.minecraft.resources.ResourceLocation;
+
+public class SimpleNetworkManager {
```

Comment:
Might be worth adding Javadocs to this down the line.

Reply:
Or even now - javadoc reduces the unnecessary guessing modders have to do when using apis.